File: Assignment1_4.py
#Accept One Number from user and check wether it is divisible by 5 or not
def ChkDivisibleBy5(iNo):
    
    if(iNo % 5 == 0):
        return "true"
    else:
        return "false"
    
def main():
    
    iValue1 = 0
    bAns = "false"
    
    print("Enter the Number:")
    iValue1 = int(input())
    
    bAns = ChkDivisibleBy5(iValue1)
    
    if(bAns == "true"):
    
        print("Number is Divisible By 5")   
    
    else:
    
        print("Not Divisible By 5")
    

if __name__=="__main__":
    main()

File: Assignment1_4.java
//Accept One Number from user and check wether it is divisible by 5 or not
import java.util.*;

class Assignment1_4
{
    public static void main(String Args[])
    {
        Scanner sobj = new Scanner(System.in);

        Checker cobj = new Checker();
        System.out.println("Enter the number..:");
        int iNo1 = sobj.nextInt();

        int iAns = cobj.DivideByFive(iNo1);

        if(iAns == 1)
        {
            System.out.println("The Number is Divisible By 5..");
        }
        else
        {
            System.out.println("The Number is Not Divisible By 5...");
        }
        sobj.close();
    }
}
class Checker
{
    public int iValue1;

    Checker()
    {
        iValue1 = 0;
    }

    int DivideByFive(int iValue1)
    {
        if(iValue1 % 5 == 0)
        {
            return 1;
        }
        else
        {
            return 0;
        }
        
    }

}